src/interceptors/response.ts

```diff
@@ -155,7 +156,10 @@ export function defaultResponseInterceptor(
         return response;
       }
 
-      ttl = expirationTime === 'not enough headers' ? ttl : expirationTime;
+      if (expirationTime !== 'not enough headers') {
```

Comment:
That would mean checking here if `typeof expirationTime === 'number'`

Reply:
It can be a number or an object